### PR TITLE
Redirect directly to FxA when login is required

### LINKED ISSUE #3205: Message about firefox account is not displayed at log in
### Steps to reproduce:
1. While logged out open the amo homepage:  https://addons-dev.allizom.org/en-US/firefox/
2. Click on the Log In link

### Expected results:
The page that explains that AMO has transitioned to Firefox Accounts is displayed

### Actual results:
No message is displayed.

### Notes/Issues:

The banner is displayed when logging in from the write a review form: 
writereview

Verified in FF47(Win7) Issue can be reproduced in amo-dev
Screencast for this issue:
directlogin

### PR DESCRIPTION
**Pull request by mstriemer**
The login required views will now redirect directly to FxA. Still not /users/login though. If you navigate there directly you'll see the "Use FxA to log in" page.

Fixes #3205.

### CODE REVIEW

File: src/olympia/accounts/helpers.py [hunk 1]
```diff
@@ -90,3 +90,7 @@ def generate_fxa_state():
 def camel_case(snake):
     parts = snake.split('_')
     return parts[0] + ''.join(part.capitalize() for part in parts[1:])
+
+
+def redirect_for_login(request):
+    return HttpResponseRedirect(default_fxa_login_url(request))
```
Comment (EnTeQuAk) [inline src/olympia/accounts/helpers.py]: So, we use the `.helpers` module usually to store template relevant code, e.g filters and jinja-functions but not general utilities (usually at least, there are some here). Can this move to `accounts.utils` instead?

**Comment by EnTeQuAk** [comment]
This is breaking a whole lot of tests.

File: src/olympia/accounts/helpers.py (hunk 1 above)
Comment (mstriemer) [inline src/olympia/accounts/helpers.py]: I find the distinction between causes a lot of circular imports. I'll try moving it though.

Comment (mstriemer) [inline src/olympia/accounts/helpers.py]: I just moved everything from `.helpers` to `.utils` and made the helpers just pull out the request object and delegate to `.utils`.

**Comment by EnTeQuAk** [comment]
r+

**Pull request merged**